### PR TITLE
AMP-69937 mParticle migration guide.

### PR DESCRIPTION
**Pull request by ppatibandla**
# Amplitude Developer Docs PR

## Description

mParticle to Amplitude CDP migration guide. I couldn't find links to groups & Data equivalent concepts in mParticle. I will update as soon as I find them. Starting PR for any early comments.

## Deadline

When do these changes need to be live on the site?
as soon as possible.

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ X] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

Couldn't get local mkdocs, will check the PR build to verify.

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/guides/mParticle-migration-guide.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+    5. [Migration checklist](#migration-checklist)
+    6. [Frequently asked questions](#frequently-asked-questions)
+
+Quickly review the offerings that are available for mParticle and how that stacks up to Amplitude. 
+
+| mParticle                                                                         | Amplitude |
+|-----------------------------------------------------------------------------------| ----------- |
+| [Connections](https://docs.mparticle.com/guides/platform-guide/connections/)      | [Sources](/data/sources/) & [Destinations](/data/destinations/) |
+| [Audiences](https://docs.mparticle.com/guides/platform-guide/audiences/overview/) | [Audiences](https://help.amplitude.com/hc/en-us/sections/360011146031-Amplitude-Audiences) |
+| N/A                                                                               | [Data Management](https://help.amplitude.com/hc/en-us/categories/5078631395227-Amplitude-CDP) |
```
Comment (ppatibandla) [inline docs/guides/mParticle-migration-guide.md:31]: Couldn't find link to equivalent concept in mParticle.

File: docs/guides/mParticle-migration-guide.md [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+|-------------------|-----------|--------------------------------------------|
+| app_key/write_key | api_key   | Unique key to validate source of the data. |
+| Workspace         | Project   | [Projects](https://help.amplitude.com/hc/en-us/articles/360058073772-Create-and-manage-organizations-and-projects#create-a-project) allow you to organize your data.  |
+| User              | User      | User who is performing action.             |
+| Identify          | Identify  | [Identify](/analytics/what-is-amplitude/#user-properties-are-details-about-your-user) updates properties/attributes of the user.|
+| logEvent          | Event     | [Event](/analytics/apis/http-v2-api-quickstart/) in Amplitude tracks the action user is performing.|
+| Screen            | Event     | Create an Event to track Screen views.|
+| Page              | Event     | Create an Event to track Page views.|
+| N/A               | Group     | [Group](/guides/accounts-instrumentation-guide/) is a collection of users. In Amplitude one user could belong to multiple groups. Each group can have properties/attributes that will be available to query/forward on actions performed by any user in the group.|
```
Comment (ppatibandla) [inline docs/guides/mParticle-migration-guide.md:60]: Same here, couldn't find link to equivalent concept in mParticle.

File: docs/guides/mParticle-migration-guide.md [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+    ``` 
+    </td>
+    </tr>
+    </table>
+ 
+For all other SDKs view the [Quickstart Guide](/data/sdks/sdk-quickstart/) and take a look at the [SDK documentation](/data/sdks/sdk-overview/).
+
+## Validate events
+
+Data validation is a critical step in the instrumentation process. Amplitude lets validate your event data via Amplitude's [User Lookup](/data/debugger/#user-lookup) or using the [Instrumentation Explorer](/data/debugger/#instrumentation-explorer) Chrome extension.
```
Comment (justin-fiedler) [inline docs/guides/mParticle-migration-guide.md:257]: Maybe also add a link to this for a full list of look-up tools https://www.docs.developers.amplitude.com/data/debugger/

File: docs/guides/mParticle-migration-guide.md [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+    ``` 
+    </td>
+    </tr>
+    <tr>
+    <td> Group </td>
+    <td>
+    </td>
+    <td>
+    ```typescript
+    groupIdentify(
```
Comment (justin-fiedler) [inline docs/guides/mParticle-migration-guide.md:119]: I think we also include `setGroup()` here

File: docs/guides/mParticle-migration-guide.md [hunk 5]
```diff
@@ -0,0 +1,299 @@
...
+
+Both mParticle and Amplitude SDKs are meant to capture first party data by tracking user interactions. For the most part both work pretty similarly except some nuances around syntax. Here is high level mapping of concepts between mParticle & Amplitude.
+
+| mParticle | Amplitude | Notes                                      |
+|----------|-----------|--------------------------------------------|
+| app_key  | api_key   | Unique key to validate source of the data. |
+| Workspace | Project   | [Projects](https://help.amplitude.com/hc/en-us/articles/360058073772-Create-and-manage-organizations-and-projects#create-a-project) allow you to organize your data.  |
+| User     | User      | User who is performing action.             |
+| Identify | Identify  | [Identify](/analytics/what-is-amplitude/#user-properties-are-details-about-your-user) updates properties/attributes of the user.|
+| logEvent | Event     | [Event](/analytics/apis/http-v2-api-quickstart/) in Amplitude tracks the action user is performing.|
```
Comment (justin-fiedler) [inline docs/guides/mParticle-migration-guide.md:57]: This is a little confusing to me. I feel it mixes the payload with methods.

I feel `logEvent` => `track`, the others make sense but I think we should be consistent to show methods or objects.

Comment (ppatibandla) [reply docs/guides/mParticle-migration-guide.md:57]: method name changes in both mparticle and our sdks based on language. How about just leaving it as Event in both?

File: docs/guides/mParticle-migration-guide.md (hunk 4 above)
Comment (ppatibandla) [reply docs/guides/mParticle-migration-guide.md:119]: You mean we need to call setGroup() before every groupIdentify?

**Comment by ppatibandla** [comment]
All the comments are applicable to both segment & mparticle. Merging this PR so we can have mParticle guide. I will create a followup PR to address these comments in both segment & mParticle guides.

**Pull request merged**